src/Microsoft.Azure.WebJobs/ResolvePropertyAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Place this on binding attributes properties to tell the binders that that the property contains 
+    /// {key} or "%appsetting%" segments that should be automatically resolved.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property)]
+    public sealed class ResolvePropertyAttribute : Attribute
```

Comment:
sounds great. 

---
In reply to: [61140559](https://github.com/Azure/azure-webjobs-sdk/pull/694#discussion_r61140559) [](ancestors = 61140559)